src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3941,6 +3941,9 @@ You should consider suppressing the warning only if you're sure that you don't w
   <data name="ERR_AttributesNotAllowed" xml:space="preserve">
     <value>Attributes are not valid in this context.</value>
   </data>
+  <data name="ERR_AttributesRequireParenthesizedLambdaExpression" xml:space="preserve">
+    <value>Attributes require a lambda expression with a parenthesized parameter list.</value>
```

Comment:
Wording of this is a bit awkward. Consider "Using attributes on a lambda expression requires a parenthesized parameter list", to avoid implying that all attributes require parenthesized lambda expressions.

Reply:
Updated to "Attributes on lambda expressions require a parenthesized parameter list."